### DISCUSSION TITLE
frigate live viewing your cameras in native GUI (no Home Assistant)

### DISCUSSION THREAD

**Discussion by stve-a** [blakeblackshear/frigate#6325] [General]
Hi all, 

I just want some clarification on whether this is possible or not.
In the Frigate native GUI, in the "cameras" view from the left side menu you see an overview of all your cameras, but the feeds are still, 0 fps.
How can I get real-time video streams in that view ?

When I click on one of the cameras I get real time live view.  (I assume this relates to the LIVE: and RTMP: options inside the config ?)
When I go to the config menu on the left, and go to the go2rtc dashboard I can see all my camera streams in real time.

My question is, can we have a live view in the cameras overview in the native GUI and if possible, which config.yml entries control its behavior ?

Thanks
Steve

**Comment by blakeblackshear** [comment]
There isn't a way to do this. That's the purpose Birdseye is supposed to serve. You can open a feature request if you want.

**Reply by stve-a** [reply]
Hi, thanks for clarifying this.  Keep up the good work !!!

**Reply by AxelHoss** [reply]
is there any update?

**Comment by builder555** [comment]
@stve-a I'm curious, what's the use case that would be different from using Birdseye?

**Reply by dylanparker** [reply]
I'll take a crack at this considering I found this thread hoping to enable live views on the dashboard page.
I'm looking to replicate the behaviour I have when viewing through AgentDVR.
I see a grid of live views (similar to BirdsEye) but I can select, then click on any individual feed and get it expanded. 
When in a grid the feeds are all sub views ... but when expanded I get the full resolution view for that camera.
AgentDVR also allows changing which camera in the grid is supplying the audio feed.
I can quickly click around the grid and listen in to the various cams ... then click again to get a high-resolution stream. A quick ESC brings me back to the grid.
I'd love to drop AgentDVR since it eats up 40% of CPU 24 hours a day just in case I'd like to view that interface.

**Reply by NickM-27** [reply]
The UI is currently being rewritten and this is how it works already 👍

**Reply by dylanparker** [reply]
Excellent! I look forward to trying it out in an upcoming release.

**Reply by tungmeister** [reply]
@NickM-27  I'm keen to build the current dev branch and have a play, some of the commits look really interesting, has anything been documented yet config wise?

**Reply by NickM-27** [reply]
@tungmeister no need to build the dev branch, builds are done automatically and available in the side bar under `packages`

The only config changes are documented in this PR https://github.com/blakeblackshear/frigate/pull/10483

**Reply by dylanparker** [reply]
I've been using a recent dev build for a few days and am quite pleased. I'll soon be able to get rid of AgentDVR completely.
Is there a design doc / discussion thread somewhere so I can better understand the long-term vision for this live interface (and possibly help contribute?)?

**Reply by NickM-27** [reply]
Once we get closer to a beta I believe there will be a post made explaining the new UI and the approach behind it. We are not really looking for feedback currently as things are still not set in stone and details are still changing and being implemented

**Reply by southerngrampiansshirecouncil** [reply]
> I see a grid of live views (similar to BirdsEye) but I can select, then click on any individual feed and get it expanded. When in a grid the feeds are all sub views ... but when expanded I get the full resolution view for that camera.

Just trying to figure out how to get this one working. I can find how to set the stream for live view, detections etc. But not set one stream for the liveview grid, and another when the camera is clicked/expanded.

Use case is for 24/7 monitoring of cameras from remotes sites with low bandwidth. Grid using a low stream minimizes the bandwidth requirements, but still allows for an operator to have the higher res full stream on demand.

**Reply by dylanparker** [reply]
> Just trying to figure out how to get this one working. I can find how to set the stream for live view, detections etc. But not set one stream for the liveview grid, and another when the camera is clicked/expanded.

To be clear, the behaviour you were replying to was behaviour I was getting with the AgentDVR software and was hoping to be implemented in Frigate. It's getting closer but I'm not entirely sure how the mix of low/high bandwidth streams are utilized with the grid/expanded interface.

**Reply by Mincka** [reply]
You need to create a group with a Birdseye camera for continuous live view of multiple cameras.
That's the closest you can get in the latest beta version.

I would like to be able to switch in group view, for each camera, between the following modes:
- Live Main Stream (continuous)
- Live Sub Stream (continuous)
- On demand (still image when no motion, and either Sub Stream or Main Stream on motion, user configurable per camera)

**Reply by southerngrampiansshirecouncil** [reply]
So just to circle back to this seeing as 14 is progressing, there is currently no way to have one/multiple grids (or with birdseye) where the grid view is a low res live sub stream and when a camera is clicked to expand the individual camera to fullscreen liveview use higher res main stream. Looking for a way to have 24/7 live grid on multiple screens (each with a subset of cameras) and minimize bandwidth requirements
